app/browser/ads/adBlockUtil.js

```diff
@@ -31,16 +31,27 @@ const mapFilterType = {
 const shouldDoAdBlockCheck = (resourceType, firstPartyUrl, url, shouldCheckMainFrame) =>
```

Comment:
yes @diracdeltas is right.  
Please do test that youtube videos do not give ads just to be sure though (logged in and out) and also do the test cases here:
https://github.com/brave/browser-laptop/commit/fc24493f2e090b23ed7ba44fee7ac7b3e8ca6f23#diff-1f92b9d4310a18761948d6ea1fbefc84